.travis.yml

```diff
@@ -0,0 +1,56 @@
+sudo: false
+language: python
+python:
+  - 3.4
+  - 3.5
+  - 3.6
+
+cache:
+  pip: true
+
+before_install:
+  - if [[ $TRAVIS_OS_NAME == osx ]]; then TRAVIS_PYTHON_VERSION=3.5; fi
+
+  - >
+    if [[ $TRAVIS_OS_NAME == osx ]]; then
+      brew install python3
+      sudo ln -sf $(which python3) $(which python)
+      sudo ln -sf $(which pip3) $(which pip 2>/dev/null || echo /usr/bin/pip)
+    fi
+
+  # Use pip 9, so that setuptools can be downgraded.
+  - pip install pip==9 setuptools==17
+
+  # https://github.com/coala/coala/issues/3183
+  # Travis automatically installs the `requirements.txt` in "install" stage
+  - cp requirements.txt requirements.orig
+  - cat test-requirements.txt >> requirements.txt
+  - sed -i.bak '/^-r/d' requirements.txt
```

Comment:
Correct, but that is needed because in that repo , `docs-requirements.txt` uses `-r..` , but here none of the requirements files use that.
So, please remove.